chapters/introduction.tex

```diff
@@ -99,14 +99,10 @@ \section{Some Definitions}\label{some-definitions}
 % - quality of implementation
 % - simulator
 
-\section{Notation and Grammar}\label{notation-and-grammar}
+\section{Notation}\label{notation}
 
-The meta symbols (of the extended BNF-grammar) are defined in~\cref{lexical-conventions}.
+In Modelica code listings, keywords are highlighted with color and may come in both normal weight and boldface, for example, \lstinline!connect! and \lstinline!equation!.
+There are also certain recognized names with special meaning defined by the specification, highlighted with another color, for example, \lstinline!semiLinear!.
+% No need to describe appearance of comments, right?
```

Comment:
Ideally, I would like to see a list of all special syntax highlighting that we use, so it can be a quick visual reference. Something like the following.

\begin{nonnormative}
This is a non-normative section that provides some explanation, motivation, and/or additional things to keep in mind.
\end{nonnormative}
\begin{example}
This is an example, which contains a description and sample code like the following:
\begin{lstlisting}[language=modelica]
model SampleCode // This is a comment.
\end{lstlisting}
\end{example}
This is inline Modelica code: \lstinline!x1 = x2!.

Modelica code highlights keywords that define code structure such as \lstinline!equation!, keywords that do not define code structure such as \lstinline!connect!, and special names defined by the specifications such as \lstinline!semiLinear!.

And so on. I actually haven't checked how the above would look after compiling. But the idea is that they are all in one place and with not too much description. So that if you are reading the specification and want to look up why something looks the way it is, you can easily match the visual with the description. Right now, the non-normative, comments and examples are described in the section above Some Definitions.

Reply:
> Right now, the non-normative, comments and examples are described in the section above Some Definitions.

Right, these two sentences look completely out of place:
> The semantic specification should be read together with the Modelica grammar.
> Non-normative text, i.e., examples and comments, are enclosed in {[}\ldots{]} and set in italics.

I moved them to two different parts of the introduction.

Reply:
> And so on. I actually haven't checked how the above would look after compiling. But the idea is that they are all in one place and with not too much description. So that if you are reading the specification and want to look up why something looks the way it is, you can easily match the visual with the description.

Sounds like a good idea to me.  It also has the benefit of not becoming out of date if we change the presentation of, say, examples.  Let's just wait a few days for other's reactions against this before including the change in the PR.

Reply:
Looks like a good idea to me.

Reply:
I've made an attempt at this now and welcome feedback.